src/tsolvers/lasolver/LASolver.cc

```diff
@@ -353,7 +353,7 @@ void LASolver::declareAtom(PTRef leq_tr)
     // DEBUG check
     isProperLeq(leq_tr);
 
-    setKnown(leq_tr);
+    setInformed(leq_tr);
```

Comment:
This can be removed. There is a call to `setInformed` on line 342.

Reply:
Removed.